### ISSUE TITLE
Base64 encoded asset

### ISSUE THREAD

**Issue by ghost** [neilime/zf2-assets-bundle#25]
Hi,

The bundle is working quite well but I've got an issue with a new asset : 

```
Fatal error: Uncaught exception 'LogicException' with message
"data:image/gif;base64,R0lGODlhCwAHAIAAACgoKP///yH5BAEAAAEALAAAAAALAAcAAAIORI4JlrqN1oMSnmmZDQUAOw=="
has not been cached as file
"path\to\cache\data:image/gif;base64,R0lGODlhCwAHAIAAACgoKP\yH5BAEAAAEALAAAAAALAAcAAAIORI4JlrqN1oMSnmmZDQUAOw=="'
 in path\to\project\vendor\neilime\zf2-assets-bundle\src\AssetsBundle\Service\Service.php on line 855 
```

Is there a built-in solution for base 64 encoded media like this gif ?

**Comment by marhub** [comment]
i have latest commit ( c6e9691800d98cb841fc724f3bbf076608792e5e ) and it's not working as expected ( file is base64 encoded png ).

**Comment by marhub** [comment]
sample css:


form fieldset.buttons .commit input { background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAAyCAYAAACd+7GKAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAClJREFUeNpi/v//vwMTAwPDfzjBgMpFI/7hFSOT9Y8qRuF3JLoHAQIMAHYtMmRA+CugAAAAAElFTkSuQmCC); }

**Comment by as-sar** [comment]
+1

**Comment by neilime** [comment]
Fixed in the version 3.4.2
